src/Windows.Core/DEVMODEW.cs

```diff
@@ -0,0 +1,783 @@
+        /// </remarks>
+        [FieldOffset(120)]
+        public uint dmDisplayFrequency;
+
+        /// <summary>
+        /// Specifies one of the <see cref="DEVMODEW_DeviceIcmMethodOptions"/> constants.
+        /// </summary>
+        [FieldOffset(124)]
+        public DEVMODEW_DeviceIcmMethodOptions dmICMMethod;
+
+        /// <summary>
+        /// Specifies one of the <see cref="DEVMODEW_DeviceIcmIntentOptions"/> constants.
+        /// </summary>
+        [FieldOffset(128)]
+        public DEVMODEW_DeviceIcmIntentOptions dmICMIntent;
+
+        /// <summary>
+        /// Specifies one of the <see cref="DEVMODEW_DeviceMediaTypeOptions"/> constants.
+        /// </summary>
+        [FieldOffset(132)]
+        public DEVMODEW_DeviceMediaTypeOptions dmMediaType;
+
+        /// <summary>
+        /// Specifies one of the <see cref="DEVMODEW_DitherTypeOptions"/> constants.
+        /// </summary>
+        [FieldOffset(136)]
+        public DEVMODEW_DitherTypeOptions dmDitherType;
+
+        /// <summary>
+        /// Not used; must be zero.
+        /// </summary>
+        [FieldOffset(140)]
+        public uint dmReserved1;
+
+        /// <summary>
+        /// Not used; must be zero.
+        /// </summary>
+        [FieldOffset(144)]
+        public uint dmReserved2;
+
+        /// <summary>
+        /// This member must be zero.
+        /// </summary>
+        [FieldOffset(150)]
+        public uint dmPanningWidth;
+
+        /// <summary>
+        /// This member must be zero.
+        /// </summary>
+        [FieldOffset(154)]
+        public uint dmPanningHeight;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="DEVMODEW"/> struct
+        /// with the <see cref="dmSize" /> field initialized.
+        /// </summary>
+        /// <returns>
+        /// An instance of the structure.
+        /// </returns>
+        public static DEVMODEW Create() => new DEVMODEW { dmSize = (ushort)sizeof(DEVMODEW) };
+    }
+
+    public enum DEVMODEW_DisplayOptions : uint
+    {
+    }
+
+    public enum DEVMODEW_PrinterOptions : uint
+    {
+        /// <summary>
+        /// The print system handles "N-up" printing.
+        /// </summary>
+        DMNUP_SYSTEM = 1,
+
+        /// <summary>
+        /// The print system does not handle "N-up" printing. An application can set <see cref="DEVMODEW.dmNup"/> to <see cref="DMNUP_ONEUP"/> if it intends to carry out "N-up" printing on its own.
+        /// </summary>
+        DMNUP_ONEUP = 2,
+    }
+
+    public enum DEVMODEW_PrinterOrientationOptions : short
+    {
+        DMORIENT_PORTRAIT = 1,
+        DMORIENT_LANDSCAPE = 2,
+    }
+
+    public enum DEVMODEW_PrintQualityOptions : short
+    {
+        DMRES_HIGH = -4,
+        DMRES_MEDIUM = -3,
+        DMRES_LOW = -2,
+        DMRES_DRAFT = -1,
+    }
+
+    public enum DEVMODEW_PrintPaperOptions : short
+    {
+        DMPAPER_FIRST = DMPAPER_LETTER,
+        DMPAPER_LETTER = 1, /* Letter 8 1/2 x 11 in */
```

Comment:
Note to self for later (unless you want to try it, @weitzhandler): use regex or multi-caret in VS to mass-convert all these comments to xml doc comments so they are exposed in the API.

Reply:
LOL
I don't mind converting them to proper XML docs so it looks more appealing C#ish.

Reply:
Thanks.
note: It's not just C# syntax that looks better. It's so that someone accessing the enum value from another project will see intellisense doc pop up with the explanation of what it means.

Reply:
Thanks for your prompt replies Andrew!

Reply:
[Here](https://bit.ly/2WkoDLL)'s how I'm converting the comments into XML docs easily. You may find it useful in the future.